source/shared/net/guides/run-node-macos.rst

```diff
@@ -0,0 +1,370 @@
+   - **Public node name**: Specify the name of your node as you want it to appear
+     on the network dashboard if the node reports to the network dashboard. *NB:
+     You must enter a name for both nodes, even if you only want to run one of them.*
+
+#. Once configured, click **Continue** to go to the *Installation Type* page.
+
+#. Click **Install** and enter your administrator password. The installer needs
+   the password because it installs the node for all users and runs it as a
+   system service.
+
+#. Once installed, click **Close**. The following helper-applications are now
+   available on your computer (you can view them in the LaunchPad or in the
+   */Applications/Concordium Node* folder):
+
+   - **Concordium Node Start Mainnet**
+
+   - **Concordium Node Stop Mainnet**
+
+   - **Concordium Node Start Testnet**
+
+   - **Concordium Node Stop Testnet**
+
+Verify that the node is running
+===============================
+
+The node runs as a background service with no user interface. To verify that
+it's running, use the **Activity Monitor**.
+
+#. Open the **Activity Monitor** application from the LaunchPad.
+
+#. Look at the list and verify that the following is running:
+
+   - ``concordium-node`` for each node you are running. For example, if you're
+     running a node on both testnet and mainnet, you'll see two instances of
+     concordium-node.
+
+   - ``node-collector`` for each node that reports to a network dashboard.
+
+#. If the node is running properly and reporting to the network dashboard, you
+   can see it on the dashboard:
+
+   - https://dashboard.mainnet.concordium.software/
+
+   - https://dashboard.testnet.concordium.com/
+
+Connect a node to the Desktop Wallet
+------------------------------------
+
+You can also verify that a node is running by connecting it to the Desktop Wallet.
+
+#. In the Desktop Wallet, go to **Settings**, and then select **Node settings**.
+
+   - If you're running the mainnet version of the Desktop Wallet, you must connect to a mainnet node. In the **Address field**, enter ``127.0.0.1`` and in the **Port field** enter ``10000``.
+
+   - If you're running the testnet version of the Desktop Wallet, you must connect to a testnet node. In the **Address field**, enter ``127.0.0.1`` and in the **Port field** enter ``10001``.
+
+#. Select **Set connection**. If the connection works and the node is running properly, there’s a message saying *Successfully connected*.
+
+Change the node startup settings
+================================
+
+If you want to change whether the node services start automatically, you have
+two options.
+
+- If you're not familiar with using a terminal, the easiest option is to reinstall the macOS node and configure it differently.
+
+- If you're familiar with using a terminal, the following
+  options are available:
+
+  - *NB: Text prefixed with a* ``$`` *is to be run in a terminal.*
+
+  - Enable automatic startup of the *node* by running:
+
+    - For Mainnet:
```

Comment:
```suggestion
    - For mainnet:
```

Reply:
Unless we start using mainnet and testnet as proper nouns like Concordium Testnet or Concordium Mainnet, we don't capitalize them.